assets/helpers/internationalisation/currency.js

```diff
@@ -21,10 +22,31 @@ export const USD: Currency = {
   glyph: '$',
 };
 
-export function forCountry(country: IsoCountry): Currency {
+function forCountry(country: IsoCountry): Currency {
   switch (country) {
     case 'US': return USD;
     case 'GB': return GBP;
     default: return GBP;
```

Comment:
This means that for the rest of the world we charge in GBP, is that correct? I thought we should be charging in USD.

Reply:
Still to be decided I think.  It is easy to change later